File: spec/code_samples/Node_request/list@cleaned@{list_name}/get.js
var request = require("request");

var options = { method: 'GET',
  url: 'https://analyzemail.com/api/v1/list/cleaned/%7Blist_name%7D',
  qs: { username: 'SOME_STRING_VALUE' },
  headers: { 'content-type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: spec/code_samples/Python_python3/list@cleaned@{list_name}/get.py
import http.client

conn = http.client.HTTPSConnection("analyzemail.com")

headers = { 'content-type': "application/json" }

conn.request("GET", "/api/v1/list/cleaned/%7Blist_name%7D?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))